environments/common/layouts/everything

```diff
@@ -82,6 +82,10 @@ openhpc
 # Hosts to run ansible-init
 cluster
 
+[compute_init:children]
+# Hosts to deploy compute initialisation ansible-init script to.
+cluster
```

Comment:
I don't think this is right, it needs to either be `builder` or `compute`. I need to discuss.